packages/marketplace/src/sagas/webhook-edit-modal.ts

```diff
@@ -231,8 +231,7 @@ export const deleteWebhook = function*({ data }: Action<DeleteWebhookParams>) {
 export const requestWebhookData = function*({ data: webhookId }: Action<string>) {
   try {
     yield put(webhookEditLoading(true))
```

Comment:
pls remove this we can set loading to action WEBHOOK_EDIT_SUBCRIPTION_REQUEST_DATA in reducer and no need to put more step to change redux state